shanoir-ng-front/src/app/shared/notifications/notifications.service.ts

```diff
@@ -42,7 +42,7 @@ export class NotificationsService {
     readonly readInterval: number = 1000;
     readonly persistenceTime: number = 1800000;
     private freshTimeouts: SuperTimeout[] = [];
-    private readonly TIMEOUT: number = 30000;
+    private readonly TIMEOUT: number = 300000;
```

Comment:
why ?

Reply:
We had problem with the timeout, sometimes it takes more than 30sc to be updated.
The other problem was that this was applied for others jobs, taht weren't download, and import always finished as "timeout" before the end of he import